Auth0/Response.swift

```diff
@@ -57,7 +57,7 @@ struct Response<E: Auth0Error> {
             return json
         }
         // This piece of code is dedicated to our friends the backend devs :)
-        if response.url?.lastPathComponent == "change_password" {
+        if response.url?.lastPathComponent == "change_password" || response.url?.lastPathComponent == "revoke" {
```

Comment:
Well ideally we shouldn't, no reason API shouldn't return JSON eg { 'success': 1 }.  Both endpoints `change_password` and `revoke` return no body (HTTP 2OO only) success, so paired it up with the original hack/workaround (otherwise it will throw as there is no content).  

In general code is set up to expect JSON, you could change this `if/else` block to return nil. The `Handlers.swift` should certainly be able to deal with it, but no doubt other potential issues?